src/NUnitFramework/tests/Constraints/UniqueItemsConstraintTests.cs

```diff
@@ -77,7 +93,17 @@ public void PerformanceTests(IEnumerable values, bool ignoreCase)
                     Assert.That(values, Is.Unique.IgnoreCase);
                 else
                     Assert.That(values, Is.Unique);
-            }, HelperConstraints.HasMaxTime(100));
+            }, HelperConstraints.HasMaxTime(150));
```

Comment:
As noted in https://github.com/nunit/nunit/pull/3502#discussion_r405096102
This was exceeding the time on CI (on my machine it only took ~24ms). It's worth noting that, with this PR, we now have to process the entire collection is order to know all non-unique values. Previously, it would failfast on the first non-unique.

Reply:
Made a couple of suggestions on this above. From memory - we added these tests as we accidentally made this too slow a few years ago - it would be great if we can avoid slowing things down as much as possible with this change. 🙂

Reply:
Similar issues on slowness: https://github.com/nunit/nunit/issues/2494 (fixed), https://github.com/nunit/nunit/issues/2799 (not fixed)